tests/step-103.sh

```diff
@@ -78,9 +83,9 @@ echo "Cloning"
 cd /tmp || exit 1
 git clone /tmp/test103 test103-clone && cd test103-clone || exit 1
 
-RESULT=$(ls -A ~/.githooks/shared 2>/dev/null)
+RESULT=$(ls -A ~/.githooks/shared 2>&1)
```

Comment:
I have a feeling the `ls -A` is breaking on Arch as that's not a POSIX flag by the looks of it.
I haven't had a chance to test it though, so I could be wrong.